meta/saiserialize.h

```diff
@@ -196,4 +199,12 @@ void sai_deserialize_free_port_oper_status_ntf(
         _In_ uint32_t count,
         _In_ sai_port_oper_status_notification_t* portoperstatus);
 
+void sai_deserialize_port_stat(
```

Comment:
look at line 192, its notification segment and you are adding enum segments here, can you move it up ?